get-info.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+name=$(</sys/class/dmi/id/product_name)
+
+dir="${0%/*}/$name"
+mkdir -p "$dir"
+
+echo "getting EDID for card0-eDP-1"
+cp /sys/class/drm/card0-eDP-1/edid "$dir"/card0-eDP-1.edid
+
+echo "getting dmesg"
+dmesg > "$dir"/dmesg
```

Comment:
As mentioned at least under Debian it requires elevated permissions. Otherwise I just get:
```
foo@bar:~$ dmesg
dmesg: read kernel buffer failed: Operation not permitted
```